### ISSUE TITLE
Intermittent error when starting lunarvim 

### ISSUE THREAD

**Issue by mhuggins7278** [0x100101/lab.nvim#11]
Occasionally when starting vim i get the following error. It happens maybe one in every 3 or 4
times

```Error executing vim.schedule lua callback: Vim:E474: Attempt to decode a blank string
stack traceback:
        [C]: in function 'json_decode'
        ...te/pack/packer/start/lab.nvim/lua/lab/quick_data_cmp.lua:33: in function ''
        vim/_editor.lua: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>```

**Comment by razak17** [comment]
I am currently facing the same issue. However, the error only occurs when I lazy load nvim-cmp. I think that is the cause the issue here.

**Comment by 0x100101** [comment]
Thanks for bringing this to my attention. The issue has been fixed in the [latest v0.2.6 release](https://github.com/0x100101/lab.nvim/releases/tag/v0.2.6).
